dts/arm/st/l4/stm32l4r5.dtsi

```diff
@@ -312,3 +312,8 @@
 		label = "OTGFS_PHY";
 	};
 };
+
+&adc1 {
+	has-temp-channel;
+	has-vref-channel;
+};
```

Comment:
Please don't add phandle fixups in this file, but rather overwrite the nodes directly:
```
		adc1: adc@foo{
			has-temp-channel;
			has-vref-channel;
		};
	};
};
```